### DISCUSSION TITLE
Новые функции

### DISCUSSION THREAD

**Discussion by sukrekus** [ferscalone/multifunctional-calculator#10] [Q&A] [answered]
Как на счёт добавления в данное приложение экономических и других финансовых функций?

**Comment by ferscalone** [comment, marked as answer]
Идея очень хорошая, с большим количеством функций. Возможно взаимодействие с внешним API. Можешь начать прописывать эти функции сам, проект открытый.

**Reply by sukrekus** [reply]
Спасибо!
